examples/evaluate_lightning_document_classification.py

```diff
@@ -0,0 +1,59 @@
+import pprint
+from pathlib import Path
+
+import typer
+from pytorch_lightning.callbacks import EarlyStopping
+
+from embeddings.defaults import RESULTS_PATH
+from embeddings.pipeline.lightning_classification import LightningClassificationPipeline
+
+
+def run(
+    embedding_name: str = typer.Option(
+        "allegro/herbert-base-cased", help="Hugging Face embedding model name or path."
```

Comment:
It should be "clarin-pl" embedding as default

Reply:
all of the other examples are using herbert from allegro

Reply:
Ok I will add this to the major refactor issue